shopinvader_product_template_multi_link/models/shopinvader_product.py

```diff
@@ -26,31 +26,36 @@ class ShopinvaderProduct(models.Model):
     @api.multi
     def _get_shopinvader_variant_link(self, link_type):
         self.ensure_one()
-        res = []
         current_backend = self.backend_id
         # Get links of the given link_type
-        for link in self.product_template_link_ids.filtered(
-            lambda x: x.link_type == link_type
-        ):
-            bindings = link.linked_product_template_id.shopinvader_bind_ids
-            for binding in bindings:
-                # Get bindings of the correct backend and lang
-                if (
-                    binding.backend_id == current_backend
-                    and binding.lang_id == self.lang_id
-                ):
-                    # Set only the "main" shopinvader variant
-                    for shopinvader_variant in binding.shopinvader_variant_ids:
-                        if shopinvader_variant.main:
-                            res.append(shopinvader_variant.id)
-                            break
-        return res
+        product_links = self.product_template_link_ids.filtered(
```

Comment:
I'd recommend to backport the current status of v13 unless you see any real problem w/ it

Reply:
I just fix errors with unit test for v10.0 (the main goal of this PR). I never use this module.
IMO this backport should be done into a new dedicated PR.